src/components/events/EventDetail/index.tsx

```diff
@@ -1,12 +1,10 @@
-import { CommunityLogo, Typography } from '@/components/common';
+import { Typography } from '@/components/common';
 import CalendarButtons from '@/components/events/CalendarButtons';
-import PointsDisplay from '@/components/events/PointsDisplay';
```

Comment:
`CommunityLogo` and `PointsDisplay` aren't being used anywhere now, so you should retire them 🫡